content/legal-notices/sub-processors/v3.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Sub-Processors
+lastUpdated: '2021-03-13T21:00:00.0Z'
+description: A list of sub-processors we use.
+---
+
+To support delivery of our services, Larder Software Limited, trading as Roadie, may engage and use data processors ("Sub-processors") with access to certain personal information.
+
+A sub-processor is an external service or provider that is enlisted by Roadie to deliver our service to you. As part of that, we may have to share personal information we have collected about you with these providers.
+
+The below table provides important information about the identity, location and role of each Sub-processor we use.
+
+| Name                     | Purpose                                                                                                | Location                 |
+| ------------------------ | ------------------------------------------------------------------------------------------------------ | ------------------------ |
+| Amazon Web Services      | Data storage provider for documents and backup of server                                               | European Union           |
+| Google, Inc              | Cloud Service Provider / Email Provider, including Google Analytics                                    | United States of America |
+| Functional Software, Inc | Analytics on application and infrastructure errors for debugging, troubleshooting, auditing, reporting | United States of America |
+| Intercom                 | Messenger for business platforms that allows customers to communcate through their app                 | United States of America |
```

Comment:
```suggestion
| Intercom R&D Unlimited Company                 | Messenger for business platforms that allows customers to communicate through their app                 | United States of America |
```

I got their official company name from their terms: https://www.intercom.com/legal/terms-and-policies

@oblongcoffee The legal entity we are dealing with is in Ireland, but Intercom data may be stored in the US. Should the location listed here be Europe or US?

Reply:
Thanks David!

Reply:
@dtuite  USA, all data is hosted in the US unless requested and paid for.